WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -49,34 +50,34 @@ extension NSNotification.Name {
     private static var zdClientId: String?
     private static var presentInController: UIViewController?
 
-    private static let sourcePlatform = "mobile_-_ios"
     private static var appVersion: String {
         return Bundle.main.shortVersionString() ?? Constants.unknownValue
     }
 
     private static var appLanguage: String {
-        guard let lang = Locale.current.languageCode else {
-            return Constants.unknownValue
-        }
-        return lang
+        return Locale.preferredLanguages[0]
     }
 
-
-
     // MARK: - Public Methods
 
     @objc static func setup() {
         guard getZendeskCredentials() == true else {
             return
         }
 
-        ZDKConfig.instance().initialize(withAppId: zdAppID,
-                                        zendeskUrl: zdUrl,
-                                        clientId: zdClientId)
+        guard let appId = zdAppID,
+            let url = zdUrl,
+            let clientId = zdClientId else {
+                DDLogInfo("Unable to set up Zendesk.")
+                toggleZendesk(enabled: false)
+                return
+        }
+
+        Zendesk.initialize(appId: appId, clientId: clientId, zendeskUrl: url)
+        Support.initialize(withZendesk: Zendesk.instance)
```

Comment:
I'm surprised the initializer takes an optional instance of `Zendesk`, which appears to be set on the preceding line.
No change requested - just an observation about the API.